SwiftStyleGuide.swift

```diff
@@ -184,6 +184,11 @@ guard 1 == 1,
 // use _ = Task<Void, Never> for task initialization, so that if a part of the
 // Task throws, the compiler forces us to handle the exception
 // More info here: https://rev.cat/catching-exceptions-from-tasks
```

Comment:
I added a few notes here so we don't fall into this again hopefully.

Reply:
great call